docs/codeql/codeql-workshops/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
You can remove this since it's in the repo root.

Reply:
done